docs/api/changelog.rst

```diff
@@ -46,6 +46,9 @@ Version 0.10
 * Fix (harmless) traceback in logs which could occur when using
   :meth:`aioxmpp.muc.Room.send_message_tracked`.
 
+* You can now register custom bookmark classes with
+  :func:`aioxmpp.bookmarks.as_bookmark_class`.
```

Comment:
Add reference to the abstract base class in both the changelog and the function.